service/acme.go

```diff
@@ -158,6 +158,9 @@ func (s *acmeService) Trigger(id string) {
 }
 
 func (s *acmeService) Renew(c *model.Certificate) {
+	if !c.Issued {
+		s.Trigger(c.ID)
```

Comment:
Does this go on to renew after starting the trigger? The Trigger is async right? Maybe it should return after calling trigger. We might also want to modify auto renew logic to check if something is issued before renewing

Reply:
oh that's weird, it's almost as if code review helps prevent stupid mistakes like this.